### PR TITLE
Add a sleep because distributed systems are hard

### PR DESCRIPTION
**Pull request by kriskowal**
r @breerly

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -17,6 +18,8 @@ func main() {
 		Behaviors: behaviors,
 	}
```
Comment (breerly) [inline main.go]: Please print "Waiting 1 second for test clients to become ready..." and then "Beginning matrix test"

File: main.go [hunk 2]
```diff
@@ -17,6 +18,8 @@ func main() {
...
 
+	time.Sleep(1 * time.Second)
```
Comment (HelloGrayson) [inline main.go]: Please print "Waiting 1 second for test clients to become ready..." and then "Beginning matrix test"

**Comment by HelloGrayson** [comment]
:shipit:

**Comment by HelloGrayson** [comment]
@kriskowal conflicts

**Comment by kriskowal** [comment]
@breerly resolved

**Pull request merged**